tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -473,8 +473,8 @@ def fit(self,
                 Hyperparameter tuning strategy and kwargs (for example, how many HPO trials to run).
                 If None, then hyperparameter tuning will not be performed.
                 Valid preset values:
```

Comment:
Maybe clearer if we replace: "Valid preset" -> "Valid str"

Reply:
I think `preset` is a parameter name here.